docs/sources/mimir/operators-guide/get-started/_index.md

```diff
@@ -9,24 +9,26 @@ weight: 10
 
 # Get started with Grafana Mimir
 
-There are two different options for getting started with Grafana Mimir:
+There are two different ways that you to get started with Grafana Mimir:
 
-- The written tutorial provides a series of imperative commands to start a single Mimir process.
-- The visual tutorial (in the form of a video) uses `docker-compose` to declaratively deploy multiple Mimir processes.
+- The written instructions that follow help you start a single Mimir process by using [_imperative_ Kubernetes commands](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/imperative-command/).<p>
```

Comment:
Note to reviewer: the terms _imperative_ and _declarative_ caused me to do research, so I added context for each. Do you agree or disagree to the accuracy of the links that I provided?